### ISSUE TITLE
Idea: Override Editor

### ISSUE THREAD

**Issue by AnthonyFuller** [atampy25/glacierkit#16]
**Is your feature request related to a problem? Please describe.**
It really sucks when you have to add overrides to a brick. You have to do it manually.

**Describe the solution you'd like**
I propose a way for you to add overrides like you were editing an entity.

Say I have an entity open and I want to override another brick. I first search for that brick, and in the context menu I can open it in an "Override Editor".
![image](https://github.com/user-attachments/assets/b9283ea1-f571-435b-b8b3-70924017307b)

This editor should have a different icon (and maybe different colour text) in the tab. Any changes I make in this editor, should be added as overrides in the original brick I had open (automatically adding this brick as an external scene). With saving working to "apply" the overrides. Of course, you can't do some things as an override, like adding entities, changing templates/blueprints, etc so these should not be allowed.

**Describe alternatives you've considered**
There's not really another way to do this other than continuing to do this manually.

**Additional context**
I can see this possibly interfering with the SDK connection.

**Comment by musicalmushr00m** [comment]
YES! This please! 
Even QNE had the overrides segmented so it was easier to organize.
A proper override editor would be very welcome.

**Comment by Ocean-Minnow** [comment]
Agreed. I never worked with overrides with QNE enough to learn enough to make them manually, but going back to QNE just to learn isn't exactly an option, for obvious reasons.
